.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+*.mmd.svg
```

Comment:
@csarven can you please suggest an alternative here? thanks!

Reply:
Don't use Mermaid + GitHub actions.